cardinal/types/debug.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+import "encoding/json"
+
+type DebugStateRequest struct{}
+
+type DebugStateElement struct {
```

Comment:
this seems identical to `CqlData` can we consolidate this into something we can use for both?

Reply:
Yeah possibly. I'll look into it.